README.md

```diff
@@ -105,6 +105,7 @@ isConnectionOpened | Sets the connection status. If set, upon reconnection smart
 onMessage | SmartCharts will send notifications via this callback, should it be provided. Each notification will have the following structure: `{ text, type, category }`.
 isAnimationEnabled | Determine whether chart animation is enabled or disabled. It may needs to be disabled for better performance. Defaults to `true`.
 showLastDigitStats | Shows last digits stats. Defaults to `false`.
+bottomWidgets | Render function for bottom widgets. Refer to [Customising Components](#customising-components).
```

Comment:
@morteza-binary could you please add it beside `topwidget` and also provide an example like we did for `topwidget`